packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1873,6 +1874,14 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return summarizeResult as ISummaryTreeWithStats;
     }
 
+    /**
+     * Implementation of IGarbageCollectionRuntime::updateStateBeforeGC.
+     * Before GC runs, called by the garbage collector to update any pending GC state.
+     */
```

Comment:
It would be great to note here what is expected from runtime when this call is made, or what runtime might want to use it for.